### PR TITLE
netlink: allow non linux builds to pass.

### PR DESCRIPTION
**Pull request by pritesh**

### CODE REVIEW

File: route_unspecified.go [hunk 1]
```diff
@@ -2,10 +2,38 @@
...
+
 func (r *Route) ListFlags() []string {
 	return []string{}
 }
 
 func (n *NexthopInfo) ListFlags() []string {
 	return []string{}
 }
+
+func (h *Handle) RouteAdd(route *Route) error {
```
Comment (aboch) [inline route_unspecified.go]: Current code organization seems to want any non implemented `func (h *Handle)` method in `handle_unspecified.go`

File: rule_unspecified.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// +build !linux
+
+package netlink
+
+func (h *Handle) RuleAdd(rule *Rule) error {
```
Comment (aboch) [inline rule_unspecified.go]: Same here. Unimplemented methods with pointer receiver `(Handle*)` must go in `handle_unspecified.go`

File: route_unspecified.go [hunk 3]
```diff
@@ -2,10 +2,38 @@
 
 package netlink
 
+import (
```
Comment (aboch) [inline route_unspecified.go]: In this file must go only unimplemented methods with pointer receivers specific to routing ex `*Route`, `*NexthoInfo`

File: rule_unspecified.go (hunk 2 above)
Comment (pritesh) [reply rule_unspecified.go]: done! moved them to handle_unspecified.go

**Comment by aboch** [comment]
Thanks. Please squash the two commits into one.

**Comment by pritesh** [comment]
done!

**Comment by aboch** [comment]
Thanks @pritesh

**Pull request merged**